theme/static/js/hs_file_browser.js

```diff
@@ -1080,19 +1080,14 @@ $(document).ready(function () {
 
     function onPaste() {
         var folderName = $("#fb-files-container li.ui-selected").children(".fb-file-name").text();
-        var targetPath = $("#hs-file-browser").attr("data-current-path");
-
-        if (folderName && folderName.lastIndexOf(".") == -1) {  // Makes sure the destination is a folder
```

Comment:
@Maurier i can check wherher the thing is a folder from the rest api.  Meanwhile right click seems to work. please try it.